x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/security/authz/70_tsdb.yml

```diff
@@ -257,6 +255,16 @@ create_doc permission can create:
                 rx: 430605511
   - match: { _version: 1 }
 
+  - do: # superuser
+      indices.refresh:
+        index: test
+  - do:
+      search:
+        index: test
+    # Original 8 docs ("setup") + 2 more (above)
+  - match: { hits.total.value: 10 }
+  - match: { hits.total.relation: "eq" }
```

Comment:
Previously these tests only verified the success/failure of the ingestion actions by the result of the index/bulk/etc request itself.
That's fine, but most other security YAML tests check the number of docs (as superuser) so I've updated these to have that same extra layer of checking.